common/templates/context.go

```diff
@@ -499,6 +499,8 @@ func baseContextFuncs(c *Context) {
 	c.addContextFunc("sendMessageNoEscapeRetID", c.tmplSendMessage(false, true))
 	c.addContextFunc("editMessage", c.tmplEditMessage(true))
 	c.addContextFunc("editMessageNoEscape", c.tmplEditMessage(false))
+	c.addContextFunc("pinMessage", c.tmplPinMessage(true))
```

Comment:
The values are 
```suggestion
	c.addContextFunc("pinMessage", c.tmplPinMessage(false))
```

The parameter specifies whether it should unpin.